packages/eas-build-job/src/index.ts

```diff
@@ -13,10 +16,12 @@ export {
   Platform,
   Cache,
 } from './common';
-export { Job, sanitizeJob } from './job';
+export { BuildJob, sanitizeBuildJob } from './job';
 export { Metadata, sanitizeMetadata } from './metadata';
 export * from './logs';
 export * as errors from './errors';
 export * from './artifacts';
 export * from './context';
 export * from './generic';
+
+export type Job = BuildJob | Generic.Job;
```

Comment:
Can we put it in `job.ts`? I mean this way we won't need to import it from `index.ts` in the files that are imported in the `index.ts` themselves.

Reply:
Moved. 👍